src/BinSkim.Rules/PERules/BA2024.EnableSpectreMitigations.cs

```diff
@@ -94,6 +94,15 @@ public IEnumerable<IOption> GetOptions()
 
         private static Dictionary<MachineFamily, CompilerVersionToMitigation[]> _compilerData = null;
 
+        // Used only for testing.
+        internal static Dictionary<MachineFamily, CompilerVersionToMitigation[]> _compilerDataCache
```

Comment:
>_compilerDataCache [](start = 81, length = 18)

why is this a property and not a field? did you do it to make this debuggable on property set?

suggest just marking _compilerData internal and be done with it :)